string data
int8 lin_vel
---
int8 result
